tests/spread/snapd-integration/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Check that the snapd/netplan integration works
+
+prepare: |
+    # put snapd in debug mode for good logs
+    mkdir -p /etc/systemd/system/snapd.service.d
+    cat <<EOF > /etc/systemd/system/snapd.service.d/local.conf
+    [Service]
+    Environment=SNAPD_DEBUG=1
+    EOF
+    systemctl daemon-reload
+    # install snapd if needed (should be installed on most lxd images)
+    if ! dpkg -l snapd >/dev/null; then
+        apt install -y snapd
+        echo "apt autoremove -y snapd" >> defered.cleanup
+    fi
+    # fake running on core (netplan only enabled on ubuntu core)
+    # FIXME: find a more offical way, this if fugly
+    cp /etc/os-release /etc/os-release.save
```

Comment:
This needs something like https://github.com/snapcore/snapd/pull/12607 to really work, this hack will just break